src/SVGPathDataTransformer.ts

```diff
@@ -13,6 +13,7 @@ import {
   intersectionUnitCircleLine,
 } from './mathUtils.js';
 import { SVGPathData } from './SVGPathData.js';
+import { REVERSE_PATH } from './transformers/reverse_path.js';
```

Comment:
this file is already long and i think its better to split it up to separate segments
ii'm planning to add few other helpers/transformers